sci-geosciences/grass/grass-8.2.0-r1.ebuild

```diff
@@ -0,0 +1,286 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
```

Comment:
Is this ready for `python3_11`?

Reply:
According to https://github.com/OSGeo/grass/blob/main/REQUIREMENTS.html GRASS support Python 3.11, but `dev-python/wxpython` not, which is a required dependency. Shall I update `PYTHON_COMPAT`, though?

Reply:
You can't then, yeah.